cinema/models.py

```diff
@@ -35,12 +39,19 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(movie: "Movie", filename: str):
+    filename = (f"{slugify(movie.title)}-{uuid.uuid4()}"
+                + pathlib.Path(filename).suffix)
+    return pathlib.Path("upload/movies/") / pathlib.Path(filename)
```

Comment:
Join `upload` and `movies` directories as well without using os-specific directory delimiters